README.md

```diff
@@ -45,3 +45,35 @@ The path to the unsigned module.
 
 ### module-out
 The path the signed module will be written to.
+
+## Usage Tips For Self-Signed Certs
+
+These steps assume that you have a JDK and OpenSSL installed. Done on OS X, but should be cross platform. You will also need Keystore Explorer. When you run 
+Keystore Explorer for the first time, you may be prompted to upgrade to unlimited strength Java crypto.
+
+To create a cert for self-signed modules, the process is roughly:
+
+1. Make a module and create a .modl file
+2. Create a keystore and certificate
+3. Open a command prompt, Change to the directory where you would like your keystore.jks file to be saved
+4. Type `keytool -genkey -alias server -keyalg RSA -keysize 2048 -keystore keystore.jks`. You will be prompted for other information needed to complete the cert. The start date of the certificate will always be today, but the end date can be changed with the -validity flag. -validity 20 will make it valid for 20 days.
+5. Open your newly created keystore.jks file with Keystore Explorer. Right-click on the server alias, and choose Export > Export Certificate Chain. In the Export dialog, change Certificate Length to Entire Chain. The export format should be PKCS#7 and PEM should be checked.
```

Comment:
The certificate chain can be exported using keytool from the command prompt instead of manually using Keystore Explorer:

```
keytool -exportcert -keystore keystore.jks -alias server -storepass <keystore password> -rfc -file <pathToMyp7b>/cert.p7b
```